src/vs/workbench/contrib/bulkEdit/browser/preview/bulkEditPane.ts

```diff
@@ -350,7 +350,7 @@ export class BulkEditPane extends ViewPane {
 			...e.editorOptions,
 			viewState: {
 				revealData: {
-					resource: { original: fileElement.edit.uri },
+					resource: { modified: fileElement.edit.uri },
```

Comment:
Afaik `edit.uri` points to the file on disk. The modified side is always the "virtual" file. So this doesn't seem good.
Can you look at the previous implementation and see how it was solved there?
If you reveal `modified`, it should really be the right uri.